docs/connector-development/config-based/tutorial/3-connecting-to-the-API-source.md

```diff
@@ -4,8 +4,7 @@ We're now ready to start implementing the connector.
 
 Over the course of this tutorial, we'll be editing a few files that were generated by the code generator:
 
-- `source-exchange-rates-tutorial/source_exchange_rates_tutorial/spec.yaml`: This is the [spec file](../../connector-specification-reference.md). It describes the inputs used to configure the connector.
```

Comment:
This file is still being generated by the auto-generator though, right? Should we remove it from the auto-generator?

Reply:
@lmossman no this is no longer generated